### DISCUSSION TITLE
Re-create deleted Queue if it was manually deleted and CRD still exists

### DISCUSSION THREAD

**Discussion by kamikaze** [rabbitmq/messaging-topology-operator#711] [Q&A] [answered]
Hi. I have manually deleted a queue via admin web UI. But its CRD still exists. I was expecting a queue to re-appear, but it doesn't. Is it a bug or are there any plans for improvements?

**Comment by Zerpet** [comment, marked as answer]
You can set time-based reconciliation, in addition to the default change-based reconciliation, using an environment variable in the Topology Operator manifest. This variable is documented here: https://rabbitmq.com/kubernetes/operator/install-topology-operator.html#parameters

In summary, you have to modify the Topology Operator manifest and add an environment variable `SYNC_PERIOD` set to a valid Go `time.Duration` string e.g. `15m` for 15 minutes. By default, this Operator does not perform time-based reconciliation.

**Comment by Elyytscha** [comment]
please set this as default, it violates default behavior of the operator pattern without this setting
